data/releases/eol.yaml

```diff
@@ -50,10 +50,6 @@ branches:
   finalPatchRelease: 1.18.20
   note: Created to solve regression introduced in 1.18.19
   release: "1.18"
-- endOfLifeDate: "2021-05-12"
-  finalPatchRelease: 1.18.19
-  note: '[Regression](https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs)'
-  release: "1.18"
```

Comment:
Removed because having two v1.18 EOL dates is unhelpful.

Reply:
@sftim To make it consistent, can you please do the same for the v1.22 release?